Package.swift

```diff
@@ -0,0 +1,74 @@
+// swift-tools-version:5.4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "ably-cocoa",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v9),
+        .tvOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "ably-cocoa",
+            targets: ["Ably"]
+        ),
+    ],
+    dependencies: [
+        .package(name: "msgpack", url: "https://github.com/rvi/msgpack-objective-C", from: "0.4.0"),
+        .package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa", .branch("main"))
+    ],
+    targets: [
+        .target(
+            name: "Ably",
```

Comment:
What's the purpose of moving SocketRocket code into Ably? They were previously a module on their own, `SocketRocket`. I've noticed you also now include `SocketRocket` headers in the Ably public header. We're losing separation of concerns/ modularity here.

Reply:
SocketRocket is now separate